deploy/fabfile.py

```diff
@@ -33,26 +32,58 @@
 home = "/home/site"
 log = "%s/dashboard.log" % home
 current = "%s/%s/dashboard" % (home, environment)
+now = time.strftime("%Y-%m-%d", time.localtime())
 
 # principal command to run upon update
-command = "cd %s && git pull && git submodule update && jekyll build >> %s" % (current, log)
+staging = "cd %s && git pull && git submodule update --remote && ./_data/import-public.rb && git add _data/projects.yml && git commit -m 'Update data for %s' && git push && ./go build >> %s" % (current, now, log)
```

Comment:
My understanding of Gemfile.lock is the same; a human should run `bundle update` and commit the resulting change. Also, I was starting to thing about adding a `./go deploy` to the Hub as well. The one thing to remember is that the `hookshot` command will need to have a `PATH=/opt/install/rbenv/shims:$PATH` prefix or some such, or you'll just have to hardcode the path to `ruby` and `bundle` where needed.